static/components/ComponentUUID_Component.js

```diff
@@ -126,7 +126,7 @@ class ComponentUUID_Component extends QR_Component{
     // }
     if(this.refs.compUuidInfo && value && value.length==36) {
```

Comment:
I don't know if you know the answer to this, but `value.length==36` begs a comment for explanation.

Reply:
No idea to be honest.  Can investigate if you want.

Reply:
The 36 refers to the total length (number of characters) in the UUID pattern: 8 + 4 + 4 + 4+ 12, plus 4 dashes.